app/views/blocked/users/index.html.erb

```diff
@@ -1,5 +1,12 @@
 <h2 class="heading"><%= t(".title") %></h2>
 
+<!--subnav-->
+<ul class="navigation actions" role="navigation">
+  <h3 class="landmark heading"><%= t("a11y.navigation") %></h3>
+  <li><%= span_if_current t(".blocked_users"), user_blocked_users_path %></li>
+  <li><%= span_if_current t(".muted_users"), user_muted_users_path %></li>
```

Comment:
The translation keys for the text of these links (and the similar ones in the muted users index) don't appear to be defined in `config/locales/views/en.yml`. I'm pretty sure they're just using the default translation (replacing underscores with spaces and capitalizing). Have you tried hovering over them in development to see if they have the "translation missing" markup?

It would be good to either define the text associated with these keys, or use different keys that are already defined. I'm not sure which is preferred in this case.